Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -19,6 +19,22 @@ import NIOConcurrencyHelpers
 import NIOCore
 import _NIODataStructures
 
+private func printError(_ string: StaticString) {
+    string.withUTF8Buffer { buf in
+        var buf = buf
+        while buf.count > 0 {
+            // 2 is stderr
+            let rc = write(2, buf.baseAddress, buf.count)
+            if rc < 0 {
+                let err = errno
+                if err == EINTR { continue }
+                fatalError("Unexpected error writing: \(err)")
+            }
+            buf = .init(rebasing: buf.dropFirst(Int(rc)))
```

Comment:
Is using `Int` here going to trip us up on some less common platform? Is it being used as a proxy to express some known number of bytes which we might be better using?